activity_browser/layouts/tabs/project_manager.py

```diff
@@ -212,6 +212,12 @@ def __init__(self, parent):
         self.header_layout.setAlignment(QtCore.Qt.AlignLeft)
         self.header_widget.setLayout(self.header_layout)
 
+        # auto-search
+        self.debounce_search = QtCore.QTimer()
+        self.debounce_search.setInterval(300)
+        self.debounce_search.setSingleShot(True)
+        self.debounce_search.timeout.connect(self.set_search_term)
```

Comment:
So what I think is breaking the tests is this interval running a Biosphere search after the project is switched to one without Biosphere installed. A quick-fix was to disable the search-test. We will need to either rewrite that test or connect within this widget to the project-changed signal like so:

`signals.project_selected.connect(self.debounce_search.stop)`